actions/init.js

```diff
@@ -3,16 +3,27 @@ const TIMEOUT = 20000;
 export const loadPropertyListings = (context, param, done) => {
     param = param || {};
     // TODO: params has hardcoded zipcode, need to change
+    const { location } = param;
     context.service.read('listings',
         { postalCode: '90034' },
```

Comment:
this is just temp right?

Reply:
yes the postalCode is temporary until we are able to get non-static responses from the api